javascript/javascript1/week4/week4.js

```diff
@@ -0,0 +1,100 @@
+const userName = [];
+const todoList = [];
+
+function getReply(command) {
+    if (command === " " || typeof command !== "string") {
+        return `Not a valid command.`;
+    }
+    else if (command.includes("my name is") && userName.length === 0) {
+        userName.push({ "name": command.split(" ").pop() });
```

Comment:
So instead of pushing to the userName array, you would assign the new value
```suggestion
        userName = command.split(" ").pop();
```

Reply:
and the rest should be changed in the similar way